doc/source/dev/toolchain.rst

```diff
@@ -254,7 +242,6 @@ OpenBLAS, ATLAS, MKL, BLIS, and reference Netlib libraries are known to work.
 =============== =====================================================
 LAPACK           3.4.1
 BLAS             A recent version of OpenBLAS, MKL or ATLAS.
-                 The Accelerate BLAS is no longer supported.
```

Comment:
This is still true, I'll put this back.

Reply:
Ah, I had not meant to imply otherwise. Just because the statement had been very old, so the "no longer" part is kinda misleading...

Reply:
NumPy re-enabled it and the question comes up regularly, so it's helpful to keep.